src/assets/defaults.ts

```diff
@@ -901,3 +907,90 @@ export const defaultSensorDataloggerProfile: OverlayGrid = {
 export const defaultDisplayUnitPreferences = {
   distance: DistanceDisplayUnit.Meters,
 }
+
+export const defaultCockpitSettings = {
```

Comment:
I liked the idea of having all the defaults centralized in one place!

One thing thought is that we need to use those in the places where they are required (e.g.: calls to `useStorage` and `useBlueOsStorage`), otherwise if someone changes the value there, the value here will be deprecated and the code will not work.